### PR TITLE
Added new credits and citations

### PR DESCRIPTION
**Pull request by notmahi**
## 👨‍🎓 Adding new credits and citations for the current works
This PR adds new credits for the VQ-BeT policies, and citation for diffusion policy, ACT/Aloha, TDMPC, and VQ-BeT. The citations were taken from the websites where applicable, and google scholar otherwise.

## How it was tested
No code changes, and thus N/A.

## How to checkout & try? (for the reviewer)
N/A.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -347,3 +348,52 @@ If you want, you can cite this work with:
...
+	booktitle={Proceedings of Robotics: Science and Systems (RSS)},
+	year={2023}
+}
+
+@article{chi2024diffusionpolicy,
+	author = {Cheng Chi and Zhenjia Xu and Siyuan Feng and Eric Cousineau and Yilun Du and Benjamin Burchfiel and Russ Tedrake and Shuran Song},
+	title ={Diffusion Policy: Visuomotor Policy Learning via Action Diffusion},
+	journal = {The International Journal of Robotics Research},
+	year = {2024},
+}
```
Comment (Cadene) [inline README.md:368]: Could we cite only one article for simplicity?

**Review by Cadene — COMMENTED** [review]
Thanks! One minor comment and let's merge

File: README.md [hunk 2]
```diff
@@ -347,3 +348,52 @@ If you want, you can cite this work with:
...
+@inproceedings{Hansen2022tdmpc,
+	title={Temporal Difference Learning for Model Predictive Control},
+	author={Nicklas Hansen and Xiaolong Wang and Hao Su},
+	booktitle={ICML},
+	year={2022}
+}
+```
+
+- VQ-BeT Policy: [VQ-BeT](https://sjlee.cc/vq-bet/)
+```
```
Comment (qgallouedec) [inline README.md:392]:
```suggestion
```bibtex
```

File: README.md [hunk 3]
```diff
@@ -347,3 +348,52 @@ If you want, you can cite this work with:
...
+  title={Learning fine-grained bimanual manipulation with low-cost hardware},
+  author={Zhao, Tony Z and Kumar, Vikash and Levine, Sergey and Finn, Chelsea},
+  journal={arXiv preprint arXiv:2304.13705},
+  year={2023}
+}
+```
+
+- [TDMPC](https://www.nicklashansen.com/td-mpc/)
+
+```
```
Comment (qgallouedec) [inline README.md:382]:
```suggestion
```bibtex
```

File: README.md [hunk 4]
```diff
@@ -347,3 +348,52 @@ If you want, you can cite this work with:
...
+
+@article{chi2024diffusionpolicy,
+	author = {Cheng Chi and Zhenjia Xu and Siyuan Feng and Eric Cousineau and Yilun Du and Benjamin Burchfiel and Russ Tedrake and Shuran Song},
+	title ={Diffusion Policy: Visuomotor Policy Learning via Action Diffusion},
+	journal = {The International Journal of Robotics Research},
+	year = {2024},
+}
+```
+- [ACT or ALOHA](https://tonyzhaozh.github.io/aloha)
+```
```
Comment (qgallouedec) [inline README.md:371]:
```suggestion
```bibtex
```

File: README.md [hunk 5]
```diff
@@ -347,3 +348,52 @@ If you want, you can cite this work with:
     year = {2024}
 }
 ```
+
+Additionally, if you are using any of the particular policy architecture, pretrained models, or datasets, it is recommended to cite the original authors of the work as they appear below:
+
+- [Diffusion Policy](https://diffusion-policy.cs.columbia.edu)
+```
```
Comment (qgallouedec) [inline README.md:355]:
```suggestion
```bibtex
```

File: README.md (hunk 1 above)
Comment (notmahi) [reply README.md]: Yup! I kept the latest version and removed the other.

**Review by Cadene — APPROVED** [review]

**Pull request merged**